pallets/assets/src/lib.rs

```diff
@@ -708,7 +708,7 @@ pub mod pallet {
 		///
 		/// Weight: `O(1)`
 		#[pallet::weight(T::WeightInfo::thaw_asset())]
-		pub(super) fn thaw_asset(
+		pub fn thaw_asset(
```

Comment:
this change may impact on the VCU bond. Please check with Ayush. I agree to remove the "super" definition.